src/components/inputs/AmountInput.js

```diff
@@ -1,20 +1,24 @@
-import React, { Component } from "react";
+import React from "react";
 import { injectIntl } from "react-intl";
 import { InputAdornment } from "@material-ui/core";
 import NumberInput from "./NumberInput";
 import { useModulesManager } from "../../helpers/modules";
 
-const AmountInput = ({ intl, ...props }) => {
+const AmountInput = ({ intl, inputMinValue = 0, ...props }) => {
```

Comment:
You can check with @dragos-dobre or @delcroip if this is a correct default value - we may have use nor negative numbers somewhere and it will be broken with that change, otherwise LGTM.

Reply:
The AmountInput is only for prices and includes currency symbols. It doesn't make sense to enter negative numbers for money.

Reply:
For entering negative values, we just use the NumberInput.